### PR TITLE
Add `BeneficiaryEvent` model and include it in `Beneficiary`

### PR DESCRIPTION
**Pull request by Publicker**

### CODE REVIEW

File: src/NoFrixion.MoneyMoov/Models/Beneficiary/Beneficiary.cs [hunk 1]
```diff
@@ -89,7 +98,7 @@ public NoFrixionProblem Validate()
 
         // Apply biz validation rules.
         var bizValidationResults = Validate(context);
-        if(bizValidationResults.Count() > 0)
+        if (bizValidationResults.Count() > 0)
```
Comment (agranillonf) [inline src/NoFrixion.MoneyMoov/Models/Beneficiary/Beneficiary.cs:101]: 💚

**Review by agranillonf — APPROVED** [review]
How do you highlight words on the title? ;O

File: src/NoFrixion.MoneyMoov/Models/Beneficiary/Beneficiary.cs (hunk 1 above)
Comment (Publicker) [reply src/NoFrixion.MoneyMoov/Models/Beneficiary/Beneficiary.cs:101]: bow

**Comment by Publicker** [comment]
> How do you highlight words on the title? ;O

@agranillonf Hhahaha, you can use any kind of markdown, just like you would in the comment section 🤘🏻

File: src/NoFrixion.MoneyMoov/Models/Beneficiary/Beneficiary.cs (hunk 1 above)
Comment (sipsorcery) [reply src/NoFrixion.MoneyMoov/Models/Beneficiary/Beneficiary.cs:101]: 💔

**Review by sipsorcery — APPROVED** [review]

**Pull request merged**